core/network-libp2p/Cargo.toml

```diff
@@ -29,6 +29,10 @@ tokio-timer = "0.2"
 unsigned-varint = { version = "0.2.1", features = ["codec"] }
 void = "1.0"
 
+slog = { version = "^2", features = ["nested-values"] }
+slog_derive = "0.1.1"
+erased-serde = "0.3.9"
```

Comment:
Why do we need this? ^^

Reply:
fyi: https://docs.rs/slog_derive/0.1.1/slog_derive/?search=#the-serdevalue-derive

Reply:
Ohh... What for a shitty derive implementation..